packages/core/deploy/047_deploy_boba_parent_messenger.js

```diff
@@ -0,0 +1,19 @@
+const func = async function (hre) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+
+  const { deployer } = await getNamedAccounts();
+
+  await deploy("Boba_ParentMessenger", {
+    contract: "Optimism_ParentMessenger",
```

Comment:
nit: might be worth a comment on these just to explain why you're deploying an "Optimism" contract on boba.